OpenRA.Mods.Common/AI/BotModules/HarvesterBotModule.cs

```diff
@@ -20,51 +20,68 @@
 
 namespace OpenRA.Mods.Common.AI
 {
-	class AIHarvesterManager
+	[Desc("Manages bot harvesters to ensure they always continue harvesting as long as there's resources on the map.")]
+	public class HarvesterBotModuleInfo : IBotModuleInfo
```

Comment:
As these are now otherwise normal traits, IMO they should live in `Mods.Common/Traits/BotModules`.